src/api/data/tiltak.ts

```diff
@@ -27,7 +27,8 @@ export const gjennomforingSchema = z.object({
 	sluttDato: nullableDateSchema,
 	status: tiltakGjennomforingStatusSchema,
 	tiltak: tiltakSchema,
-	arrangor: arrangorSchema
+	arrangor: arrangorSchema,
+	koordinatorer: z.array(z.string())
```

Comment:
Jeg er litt usikker på om det er ønskelig å legge denne på standard gjennomføring schemaet, siden liste over koordinatorer er en spesiell ting som kun 1 visning trenger. Hadde det vært bedre med eget endepunkt og DTO for å hente koordinatorer?

Reply:
hmm, kanskje. Jeg tenker jo at det å minimere antall kall mot backend er en god ting (om det ikke går ut over hastigheten) så man har mindre feilkilder, men kan være enig i at dette kanskje kan være et eget skjema i frontenden... usikker på hva jeg tenker der egentlig.